config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+    tags_controller:
+      tagstitle: "Etichette"
+    users_controller:
+      usertitle: "Utenti"
+    signup_controller:
+      signuptitle: "Condividi un invito a tua volta"
+      signedup: "Siete già iscritti."
+      invalidinv: "Invito invalido o scaduto."
+      flashsuccesssignupcontroller: "Benvenuto!"
+  # suggest.stories scope, unable to get it so global, I don't know why
+  suggeststorychanges: "Suggerire delle modifiche all'articolo"
+  suggestchangesbutton: "Suggerire delle modifiche"
+  suggestor: "o"
+  suggestcancel: "Annullare"
+  # reply at in views/email_message/notification.text.erb
+  replyat: "Rispondi a "
+  # reply at in views/email_reply/reply.text.erb
+  emailreplyat: "%{author} ti ha risposto: "
+  emailcontinueto: "Continua la discussione su"
+  # reply at in views/email_reply/mention.text.erb
+  emailmentionreplyat: "%{author} ti ha citato in un commento:"
+  emailmentioncontinueto: "Continua la discussione su"
+  date:
+    abbr_day_names:
+    - dom
+    - lun
+    - mar
+    - mer
+    - gio
+    - ven
+    - sab
+    abbr_month_names:
+    -
+    - gen.
+    - feb.
+    - mar.
+    - apr.
+    - mag
+    - giu
+    - lug.
+    - ago
+    - set.
+    - ott.
+    - nov.
+    - dic.
+    day_names:
+    - domenica
+    - lunedì
+    - martedì
+    - mercoledì
+    - giovedì
+    - venerdì
+    - sabato
+    formats:
+      default: "%d/%m/%Y"
+      short: "%e %b"
+      long: "%e %B %Y"
+    month_names:
+    - gennaio
+    - febbraio
+    - marzo
+    - aprile
+    - maggio
+    - giugno
+    - luglio
+    - agosto
+    - settembre
+    - ottobre
+    - novembre
+    - dicembre
+    order:
+    - :day
+    - :month
+    - :year
+  datetime:
+    distance_in_words:
+      about_x_seconds:
+        one: un secondo fa
+        other: "%{count} secondi fa"
+      about_x_minutes:
+        one: un minuto fa
+        other: "%{count} minuti fa"
+      about_x_hours:
+        one: "un'ora fa"
+        other: "%{count} ore fa"
+      about_x_months:
+        one: un mese fa
+        other: "%{count} mesi fa"
+      about_x_years:
+        one: un anno fa
+        other: "%{count} anni fa"
+      almost_x_years:
+        one: quasi un anno
+        other: quasi %{count} anni
+      half_a_minute: mezzo minuto
+      less_than_x_minutes:
+        zero: meno di un minuto
+        one: meno di un minuto
+        other: meno di %{count} minuti
+      less_than_x_seconds:
+        zero: meno di un secondo
+        one: meno di un secondo
+        other: meno di %{count} secondi
+      over_x_years:
+        one: più di un anno
+        other: più di %{count} anni
+      x_days:
+        one: 1 giorno fa
+        other: "%{count} giorni fa"
+      x_minutes:
+        one: 1 minuto fa
+        other: "%{count} minuti fa"
+      x_months:
+        one: 1 mese fa
+        other: "%{count} mesi fa"
+      x_seconds:
+        one: 1 secondo fa
```

Comment:
1 -> un ?

Reply:
È rilevante?

Reply:
in Italiano usare il numero qui sarebbe sbagliato, allo stesso tempo a livello visuale secondo me i numeri sono meglio

Reply:
I numeri danno un colpo d'occhio migliore